pygeoogc/__init__.py,sha256=iJKom17JmQSnCPIKTnPckxFB1j0Lnlqzq4TvdoKFH1U,421
pygeoogc/exceptions.py,sha256=moWPK_rK2jWWB3HI7eB_U9oPhKoG107--4osAZnAZeg,1969
pygeoogc/pygeoogc.py,sha256=6SnEsC3J08CaEX6__FZa_lkGXszFEOgz7uJFLkb-wd4,26588
pygeoogc/utils.py,sha256=3_z2sYwM6vLTMZlJW0220V3GLsbwZk42YfxHvAIwLCo,10105
pygeoogc-0.1.3.dist-info/AUTHORS.rst,sha256=FAneYiHTTZuEAtX7AvBFFp5_QesoMSH0cUAt3tirVYQ,157
pygeoogc-0.1.3.dist-info/LICENSE,sha256=EM87NgYgabO0a_E-z2nmbsrxomXdI5BDCgmomVDvLsE,1096
pygeoogc-0.1.3.dist-info/METADATA,sha256=9aXB7Pd6ikb93eyf2dhdBpbmXaDS89qZXaJBaNp30M0,5587
pygeoogc-0.1.3.dist-info/WHEEL,sha256=kGT74LWyRUZrL4VgLh6_g12IeVl_9u9ZVhadrgXZUEY,110
pygeoogc-0.1.3.dist-info/top_level.txt,sha256=LlCRBzIb5C1PpXavuTkqP7BhlJHJARnYgOzJFpXmxms,9
pygeoogc-0.1.3.dist-info/RECORD,,
